lib/rack/utils.rb

```diff
@@ -140,6 +140,21 @@ def q_values(q_value_header)
       end
     end
 
+    FORWARDED_PAIR_REGEX = /\A\s*(by|for|host|proto)\s*=\s*"?([^"]+)"?\s*\Z/i
```

Comment:
Should we make this private?

Reply:
That's a good idea, I'll make that change.

Reply:
This constant is only used in one place, so I inlined it instead.